eng/Version.Details.xml

```diff
@@ -58,9 +58,9 @@
       <Uri>https://github.com/dotnet/corefx</Uri>
       <Sha>bedb348c4d93b6172967df6cdeea0272c81eb752</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.App.Ref" Version="3.0.1">
+    <Dependency Name="Microsoft.NETCore.App.Ref" Version="3.0.0" Pinned="true">
```

Comment:
Why is CoreFx building a 3.0.1 version of this package at all. Seems like building ref packages and targeting packs that include them should be explicitly opt in across our repos.

@aspnet/build (@JunTaoLuo and @wtgodbe in particular) please check to ensure Microsoft.Internal.Extensions.Refs packages aren't being created. I believe we're currently building Microsoft.AspNetCore.App.Ref and (due to a typo in one of my 3.0.1-related PRs). I'm busy elsewhere and would appreciate someone else fixing https://github.com/aspnet/AspNetCore/blob/release/3.0/src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj#L81 (should be an OR) and whatever else I busted.

Reply:
@dougbu It's a good question, but this is at least a temporary remedy to avoid accidetnally upgrading.

Reply:
I just confirmed we're not building `Microsoft.Internal.Extensions.Refs` in 3.0 anymore. I'll put up a PR to disable building `Microsoft.AspNetCore.App.Ref`.

Reply:
@nguerrera didn't we conclude that we DO want to build and ship Microsoft.AspNetCore.App.Ref 3.0.1? This is how core-setup picks up our fix for the targeting pack.

Reply:
In this case, we do need a 3.0.1 targeting pack to fix a bug in the 3.0.0 targeting pack. Targeting packs can be acquired from nuget so any time we change them post-release (hopefully rarely), and we do need to build them and we do need a higher number. The content inside the 3.0.1 pack should have the bug fix and only the bug fix. No assembly version changes. No real surface area changes.

Reply:
The way to think of it is an increase in targeting pack patch version means a patch to the representation of the major.minor TFM, it still represents the entirety of the TFM. You should be able to build with a patched targeting pack and run on an unpatched runtime.

Reply:
My point was this should be opt-in. Yes, of course, we want to opt in very soon now.

In addition, current builds AspNetCore builds produce horribad copies of the ref/ assemblies because they don't have the right versions. Should never expose broken stuff.

Reply:
@nguerrera Are you implying that if we ship an updated aspnetcore ref pack, we need to do the same for M.N.A ref pack?

Reply:
I am NOT implying that.